src/plugins/columnSorting/test/columnSorting.e2e.js

```diff
@@ -1,11 +1,11 @@
 describe('ColumnSorting', () => {
-  var id = 'testContainer';
+  const id = 'testContainer';
 
   beforeEach(function() {
     this.$container = $(`<div id="${id}" style="overflow: auto; width: 300px; height: 200px;"></div>`).appendTo('body');
 
-    this.sortByColumn = function(columnIndex) {
-      var element = this.$container.find(`th span.columnSorting:eq(${columnIndex})`);
+    this.sort = function(columnIndex) {
```

Comment:
Maybe `sortByColumnHeader` would be more precise? The plugin has a method with the same name and it could be confusing.